landing has been done

MyNovikov opened the pull request:
[DEMO LINK](https://MyNovikov.github.io/Kickstarter/)

vadiimvooo reviewed (changes requested):
Good job. You have to make some fixes:
1. Add hover effect for interactive elements (links, buttons, images) on your page.
2. You should increase font-weight only on hover
![image](https://github.com/mate-academy/Kickstarter/assets/108701488/efb94579-c950-424c-abc9-b5e318bb6321)
3. This block should be reworked, please look at the mockup.
![image](https://github.com/mate-academy/Kickstarter/assets/108701488/178d1c47-cefc-4f78-be27-016580c15311)
4. As well as this section
![image](https://github.com/mate-academy/Kickstarter/assets/108701488/360c5f02-0f38-4463-84d1-b9525d9a3d60)
5. Layout of this section is also incorrect:
![image](https://github.com/mate-academy/Kickstarter/assets/108701488/543ec0d5-621d-4cda-ba09-6f9181bd3be5)
6. Form should reset on submit.
7. Layout version of slider should be also changed according to mockup
![image](https://github.com/mate-academy/Kickstarter/assets/108701488/f3210861-3cca-4070-ab0b-dcaf4c2a9e21)
8. On mobile version horizontal scroll appears. You have to fix it.
![image](https://github.com/mate-academy/Kickstarter/assets/108701488/a490c4fd-10d9-433d-9d92-03f96ef92723)

SanyaBratashchuk reviewed (changes requested):
1. Do not change link font-size on hover to avoid content jumping
![image](https://github.com/mate-academy/Kickstarter/assets/97547824/500a5c4c-7e3a-49e1-94e4-f470d54c4800)
2. add favicon
3. add correct page title

vadiimvooo reviewed (changes requested):
Problem with jumping links still not fixed, You have to remove font-weight on hover styles for them. Please ask for help in chat if you have some problems with the task.

MyNovikov commented:
> Problem with jumping links still not fixed, You have to remove font-weight on hover styles for them. Please ask for help in chat if you have some problems with the task.

@vadiimvooo, pls provide a screenshot or video of what exactly is wrong. I removed the font-size for those links that were on the screenshot. Left only font-weight, аccording to the design. And I added a "scale" from myself, because the font-weight does not change smoothly. Аnd with the help of "scale" font-weight seems to be smooth. Anyway, I add another fix

maxim2310 reviewed (approved):
Good work!!!
